src/test/java/org/jolokia/docker/maven/assembly/DockerFileBuilderTest.java

```diff
@@ -26,68 +25,89 @@ public void testBuildDockerFile() throws Exception {
                 .maintainer("maintainer@example.com")
                 .workdir("/tmp")
                 .labels(ImmutableMap.of("com.acme.foobar", "How are \"you\" ?"))
-                .volumes(Collections.singletonList("/vol1")).content();
+                .volumes(Collections.singletonList("/vol1"))
+                .runCommands(Arrays.asList("echo something","echo second"))
+                .content();
 
         String expected = loadFile("docker/Dockerfile.test");
         assertEquals(expected, stripCR(dockerfileContent));
     }
 
-    @Test
-    public void testMaintainer() {
-        String dockerfileContent = new DockerFileBuilder().maintainer("maintainer@example.com").content();
-        assertThat(dockerfileToMap(dockerfileContent), hasEntry("MAINTAINER", "maintainer@example.com"));
-    }
+	@Test
+	public void testMaintainer() {
+		String dockerfileContent = new DockerFileBuilder().maintainer(
+				"maintainer@example.com").content();
+		assertThat(dockerfileToMap(dockerfileContent),
+				hasEntry("MAINTAINER", "maintainer@example.com"));
+	}
 
-    @Test
-    public void testEntryPointShell() {
-        Arguments a = Arguments.Builder.get().withShell("java -jar /my-app-1.1.1.jar server").build();
-        String dockerfileContent = new DockerFileBuilder().entryPoint(a).content();
-        assertThat(dockerfileToMap(dockerfileContent), hasEntry("ENTRYPOINT", "java -jar /my-app-1.1.1.jar server"));
-    }
+	@Test
+	public void testEntryPointShell() {
+		Arguments a = Arguments.Builder.get()
+				.withShell("java -jar /my-app-1.1.1.jar server").build();
+		String dockerfileContent = new DockerFileBuilder().entryPoint(a)
+				.content();
+		assertThat(dockerfileToMap(dockerfileContent),
+				hasEntry("ENTRYPOINT", "java -jar /my-app-1.1.1.jar server"));
+	}
 
-    @Test
-    public void testEntryPointParams() {
-        Arguments a = Arguments.Builder.get().withParam("java").withParam("-jar").withParam("/my-app-1.1.1.jar").withParam("server").build();
-        String dockerfileContent = new DockerFileBuilder().entryPoint(a).content();
-        assertThat(dockerfileToMap(dockerfileContent), hasEntry("ENTRYPOINT", "[\"java\",\"-jar\",\"/my-app-1.1.1.jar\",\"server\"]"));
-    }
+	@Test
+	public void testEntryPointParams() {
+		Arguments a = Arguments.Builder.get().withParam("java")
+				.withParam("-jar").withParam("/my-app-1.1.1.jar")
+				.withParam("server").build();
+		String dockerfileContent = new DockerFileBuilder().entryPoint(a)
+				.content();
+		assertThat(
+				dockerfileToMap(dockerfileContent),
+				hasEntry("ENTRYPOINT",
+						"[\"java\",\"-jar\",\"/my-app-1.1.1.jar\",\"server\"]"));
+	}
 
-    @Test
-    public void testNoRootExport() {
-        assertFalse(new DockerFileBuilder().add("/src", "/dest").basedir("/").content().contains("VOLUME"));
-    }
+	@Test
```

Comment:
i am sorry about this, but it is code from @javapapo 
i only did some work ahead of it to push it to a pull request
Please reformat as needed. Thank you.